boards/arm/sam_e70_xplained/doc/sam_e70_xplained.rst

```diff
@@ -141,6 +141,17 @@ Flashing
    The command will also verify that the image was programmed correctly, reset
    the board and run the Zephyr application.
 
+   It is also possible to use an external debug adapter such as J-Link or ULINK
```

Comment:
Reads better to get the subject earlier in the sentence:

You can flash the image using an external debug adapter such as J-Link or ULINK,  connected to the
20-pin JTAG header.